### PR TITLE
Pass results from security analysis to aggrid

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

File: src/components/security-analysis-result.js [hunk 1]
```diff
@@ -134,19 +154,19 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
                 postContingencyResult.status !== 'CONVERGED'
             ) {
                 rows.push({
-                    contingencyIndex: index,
+                    //contingencyIndex: index,
```
Comment (YenguiSeddik) [inline src/components/security-analysis-result.js:157]: Remove this comment if the field is not used anymore

File: src/components/security-analysis-result.js [hunk 2]
```diff
@@ -134,19 +154,19 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
...
                     contingencyId: postContingencyResult.contingency.id,
                     computationStatus: postContingencyResult.status,
                     violationCount:
                         postContingencyResult.limitViolationsResult
                             .limitViolations.length,
-                    _group: index,
-                    _root: true,
+                    //  _group: index,
+                    //  _root: true,
                 });
```
Comment (YenguiSeddik) [inline src/components/security-analysis-result.js:165]: same here and all the commented field below

**Review by YenguiSeddik — CHANGES_REQUESTED** [review]

**Review by anistouri — APPROVED** [review]

File: src/components/security-analysis-result.js (hunk 1 above)
Comment (AAJELLAL) [reply src/components/security-analysis-result.js]: Done

File: src/components/security-analysis-result.js (hunk 2 above)
Comment (AAJELLAL) [reply src/components/security-analysis-result.js:162]: Done

File: src/components/security-analysis-result.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (AAJELLAL) [reply src/components/security-analysis-result.js]: Done

File: src/components/security-analysis-result.js [hunk 3]
```diff
@@ -134,19 +154,19 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
...
                 postContingencyResult.limitViolationsResult.limitViolations.forEach(
                     (limitViolation) => {
                         rows.push({
-                            contingencyIndex: index,
+                            //   contingencyIndex: index,
```
[the remark this answers is not in the document]
Comment (AAJELLAL) [reply src/components/security-analysis-result.js]: Done

File: src/components/security-analysis-result.js [hunk 4]
```diff
@@ -155,7 +175,9 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
                             value: limitViolation.value,
                             loading: computeLoading(limitViolation),
                             side: limitViolation.side,
-                            _group: index,
+                            //     _group: index,
```
[the remark this answers is not in the document]
Comment (AAJELLAL) [reply src/components/security-analysis-result.js]: Done

**Review by YenguiSeddik — APPROVED** [review]

**Review by jonenst — APPROVED** [review]

**Pull request merged**